test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -182,10 +182,15 @@ class ExtAuthzGrpcIntegrationTest : public Grpc::VersionedGrpcClientIntegrationP
     }
 
     if (!new_headers_from_upstream.empty()) {
-      // new_headers_from_upstream has append = true. The current implementation ignores to set
-      // multiple headers that are not present in the original request headers. In order to add
-      // headers with the same key multiple times, setting response headers with append = false and
-      // append = true is required.
+      // new_headers_from_upstream has the `append=true` flag set, which tells Envoy to append to
+      // the existing header if it exists, and to discard the insertion otherwise. There's a
+      // "TODO(dio)" in ext_authz.cc to add a flag to make it be "append-if-set or set-if-unset",
```

Comment:
Probably rephrase this to: In _method_name_ ...

And expand `ext_authz.cc` to full relative-path from root: source/extensions/filters/http/ext_authz/ext_authz.cc?

And I'm not sure a "TODO(dio)" is needed here. A pending task or a TODO is enough?

Reply:
Whoops, I didn't think about that that reference would tag you.  My bad.

How's this?
```
// the existing header if it exists, and to discard the insertion otherwise. In
// `Filter::onComplete()` in `ext_authz.cc`, there's a "todo" to add a flag to make it be
// "append-if-set or set-if-unset", but for now it's just "set-if-unset".
```